pkg/pipeline/conntrack/conntrack.go

```diff
@@ -141,14 +142,20 @@ func (ct *conntrackImpl) Track(flowLogs []config.GenericMap) []config.GenericMap
 			ct.connStore.addConnection(computedHash.hashTotal, conn)
 			ct.updateConnection(conn, fl, computedHash)
 			if ct.shouldOutputNewConnection {
-				outputRecords = append(outputRecords, conn.toGenericMap())
+				record := conn.toGenericMap()
+				addHashField(record, computedHash.hashTotal)
```

Comment:
@ronensc what happened to the famous "builder" methodology ( consider making the code a bit nicer with such  :-)   )

Reply:
I think the builder design pattern is a bit overkill in this use case